Formula/ale.rb

```diff
@@ -0,0 +1,39 @@
+class Ale < Formula
+  desc "The Arcade Learning Environment (ALE) -- a platform for AI research."
+  homepage "https://github.com/mgbellemare/Arcade-Learning-Environment"
+  url "https://github.com/mgbellemare/Arcade-Learning-Environment/archive/v0.5.2.tar.gz"
+  sha256 "bc158b4fb8edc5034f8f8b4ca1a90d48590f516b33f9edec325fce8762418069"
+
+  head url "https://github.com/mgbellemare/Arcade-Learning-Environment.git"
```

Comment:
Isn't the cause of CI failure, but as Baptiste pointed out earlier this should just be:
```ruby
head "https://github.com/mgbellemare/Arcade-Learning-Environment.git"
```

Reply:
Ok, let me make this change and see if it makes things better.